pysqldb3/tests/test_backup_table.py

```diff
@@ -278,6 +278,60 @@ def test_messy_table(self):
         if os.path.isfile(test_back_file):
             os.remove(test_back_file)
         assert not os.path.isfile(test_back_file)
+
+    def test_backup_tables_capitalized(self):
+        db.drop_table(table=test_pg_to_backup.capitalize(), schema=pg_schema)
+
+        # table schema
+        db.query(f"""
+            CREATE TABLE {pg_schema}.{test_pg_to_backup.capitalize()} (
+                id int,
+                column2 text,
+                column3 timestamp,
+                "1 test messy column" text
+            );
+        """)
+        # populate table
+        for i in range(10):
+            db.query(f"""
+                INSERT INTO {pg_schema}.{test_pg_to_backup.capitalize()}
+                    (id, column2, column3, "1 test messy column")
+                values ({i}, '{i}', '{'2022-10-14 10:12:40-04'}', '{'test '*i}')
+            """)
+        # validate table created
+        assert db.table_exists(test_pg_to_backup.capitalize(), schema=pg_schema)
```

Comment:
this will also currently fail (which is fine for now) since table exists doesnt account for capitalized names, you should probably just recreate the underlying sql logic to check if table exists and account for capitalized name

Reply:
I know you made #79 to address the capitalization in `table_exists` and it looks like it works. I had copied over the code manually into my local version, tested the backup tables with the revised `table_exists` logic, and then made sure not to upload the `table_exists` code change since it'll change once #79 gets merged.